proving/proving_test.go

```diff
@@ -66,28 +65,37 @@ func Test_Generate(t *testing.T) {
 			r.NoError(err)
 			r.Equal(len(ch), n)
 
-			proof, proofMetaData, err := Generate(context.Background(), ch, cfg, log, WithDataSource(cfg, nodeId, commitmentAtxId, opts.DataDir))
+			proof, proofMetaData, err := Generate(
+				context.Background(),
+				ch,
+				cfg,
+				log,
+				WithDataSource(cfg, nodeId, commitmentAtxId, opts.DataDir),
+				WithPowScryptParams(config.DefaultPowScryptParams()))
```

Comment:
Super-NIT 😄 : I don't know why gofmt doesn't do this automatically:

```suggestion
				WithPowScryptParams(config.DefaultPowScryptParams()),
			)
```

imo leads to better visual signalling to what is part of the parameter list and if you add or remove a parameter only one line shows up in the diff (instead of two if you add or remove one at the end of the list).